pipelinewise/cli/alert_handlers/victorops_alert_handler.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, config: dict) -> None:
             raise InvalidAlertHandlerException('No valid VictorOps config supplied.')
 
     def send(
-        self, message: str, level: str = BaseAlertHandler.ERROR, exc: Exception = None
+        self, message: str, level: str = BaseAlertHandler.ERROR, exc: Exception = None, **kwargs
```

Comment:
is `**kwargs` needed here to "hide" the not victorops relevant `tap_slack_channel` argument?

Reply:
Yes it is.